client/service/src/client/client.rs

```diff
@@ -106,6 +106,7 @@ where
 	executor: E,
 	storage_notifications: StorageNotifications<Block>,
 	import_notification_sinks: NotificationSinks<BlockImportNotification<Block>>,
+	every_block_import_notification_sinks: NotificationSinks<BlockImportNotification<Block>>,
```

Comment:
```suggestion
	every_import_notification_sinks: NotificationSinks<BlockImportNotification<Block>>,
```

by analogy with `every_import_notification_stream`. otherwise you can rename `every_import_notification_stream` to `every_block_import_notification_stream`